.github/workflows/fast_testing.yml

```diff
@@ -0,0 +1,49 @@
+name: fast_testing
```

Comment:
In the commit message you said: `Add new .github/workflow/precommit_testing.yml` but the patch does not add such file. It adds `.github/workflows/fast_testing.yml`.

Reply:
Oops, right, fixed. It should be 'fast_testing' everywhere.